web/src/features/AppVersionHistory/AppVersionHistoryRow.tsx

```diff
@@ -559,11 +595,7 @@ class AppVersionHistoryRow extends Component {
               className="status-tag success flex-auto u-cursor--default"
               data-tip={
                 version.deployedAt
-                  ? `${
-                      version.status === "deploying"
```

Comment:
`version.status` will never be deploying in this block because of this check: 
https://github.com/replicatedhq/kots/blob/ec4b3a55fd79a5c9758025b800f3c72bc3756a27/web/src/features/AppVersionHistory/AppVersionHistoryRow.tsx#L591

also `version.status === "deploying"` get handled here: 

https://github.com/replicatedhq/kots/blob/ec4b3a55fd79a5c9758025b800f3c72bc3756a27/web/src/features/AppVersionHistory/AppVersionHistoryRow.tsx#L640